biom/cli/table_summarizer.py

```diff
@@ -23,7 +23,7 @@ def summarize_table(table, qualitative=False, observations=False):
         compute_counts_per_sample_stats(table, qualitative)
     num_observations = len(table.ids(axis='observation'))
 
-    counts_per_sample_values = counts_per_samp.values()
+    counts_per_sample_values = list(counts_per_samp.values())
```

Comment:
That's a dict
On Oct 20, 2015 12:51 PM, "Evan Bolyen" <notifications@github.com> wrote:

> In biom/cli/table_summarizer.py
> <https://github.com/biocore/biom-format/pull/661#discussion_r42537523>:
>
> > @@ -23,7 +23,7 @@ def summarize_table(table, qualitative=False, observations=False):
> >          compute_counts_per_sample_stats(table, qualitative)
> >      num_observations = len(table.ids(axis='observation'))
> >
> > -    counts_per_sample_values = counts_per_samp.values()
> > +    counts_per_sample_values = list(counts_per_samp.values())
>
> If this is an ndarray .tolist() is much faster than using the list
> constructor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/biom-format/pull/661/files#r42537523>.
>